src/components/Dotting.tsx

```diff
@@ -30,14 +30,17 @@ export interface DottingProps {
   gridStrokeWidth?: number;
   isGridVisible?: boolean;
   backgroundMode?: "checkerboard" | "color";
-  backgroundColor?: React.CSSProperties["color"];
+  backgroundColor?: string;
```

Comment:
is regular string better than `React.CSSProperties["color"]`?

Reply:
I tried using react.cssproperties but it seems that stories.tsx does not interpret it well (when cssproperties is used, there is no color select option for the dotting component stories page) If string is used, the docs allows you to change the color of the property.